src/utils.ts

```diff
@@ -130,3 +131,20 @@ export function getActiveElement(doc?: Document): Element | null {
     return doc.body;
   }
 }
+
+export function warnRenderProps(c: string, props: any, ignored: string) {
```

Comment:
This is actually inconsistent between `<Formik />` and `<Field />, <FastField />, etc.` - in `Formik` a `component` prop takes precedence and in `Field` a `render` prop takes precedence. Could we pick one (even if types are changed to eliminate this as a problem in TS).